src/Site_WP_Command.php

```diff
@@ -191,27 +183,27 @@ public function create( $args, $assoc_args ) {
 		EE::warning( 'This is a beta version. Please don\'t use it in production.' );
 		$this->logger->debug( 'args:', $args );
 		$this->logger->debug( 'assoc_args:', empty( $assoc_args ) ? array( 'NULL' ) : $assoc_args );
-		$this->site['name'] = strtolower( EE\Utils\remove_trailing_slash( $args[0] ) );
+		$this->site['url'] = strtolower( EE\Utils\remove_trailing_slash( $args[0] ) );
 
 		$mu = EE\Utils\get_flag_value( $assoc_args, 'mu' );
 
 		if ( isset( $assoc_args['mu'] ) && ! in_array( $mu, [ 'subdom', 'subdir' ], true ) ) {
 			EE::error( "Unrecognized multi-site parameter: $mu. Only `--mu=subdom` and `--mu=subdir` are supported." );
 		}
-		$this->site['type'] = $mu ?? 'wp';
+		$this->site['app_sub_type'] = $mu ?? 'wp';
 
-		if ( EE::db()::site_in_db( $this->site['name'] ) ) {
-			EE::error( sprintf( "Site %1\$s already exists. If you want to re-create it please delete the older one using:\n`ee site delete %1\$s`", $this->site['name'] ) );
+		if ( Site::find( $this->site['url'] ) ) {
+			EE::error( sprintf( "Site %1\$s already exists. If you want to re-create it please delete the older one using:\n`ee site delete %1\$s`", $this->site['url'] ) );
 		}
 
-		$this->proxy_type      = EE_PROXY_TYPE;
 		$this->cache_type      = EE\Utils\get_flag_value( $assoc_args, 'cache' );
 		$this->le              = EE\Utils\get_flag_value( $assoc_args, 'letsencrypt' );
-		$this->site['title']   = EE\Utils\get_flag_value( $assoc_args, 'title', $this->site['name'] );
-		$this->site['user']    = EE\Utils\get_flag_value( $assoc_args, 'admin_user', 'admin' );
-		$this->site['pass']    = EE\Utils\get_flag_value( $assoc_args, 'admin_pass', EE\Utils\random_password() );
-		$this->db['name']      = str_replace( [ '.', '-' ], '_', $this->site['name'] );
+		$this->site['title']   = EE\Utils\get_flag_value( $assoc_args, 'title', $this->site['url'] );
+		$this->site['wp_user']    = EE\Utils\get_flag_value( $assoc_args, 'admin_user', 'admin' );
+		$this->site['wp_pass']    = EE\Utils\get_flag_value( $assoc_args, 'admin_pass', EE\Utils\random_password() );
+		$this->db['name']      = str_replace( [ '.', '-' ], '_', $this->site['url'] );
 		$this->db['host']      = EE\Utils\get_flag_value( $assoc_args, 'dbhost' );
+		//TODO: Generate db user according to site
```

Comment:
Do we have any other task to track this? If not please create one.

Reply:
Created - https://github.com/EasyEngine/site-wp-command/issues/18